bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/datalayer/DataLayerComponent.java

```diff
@@ -15,23 +15,28 @@
 
 import javax.inject.Inject;
 
-import org.apache.commons.codec.digest.DigestUtils;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.sling.api.resource.Resource;
 import org.apache.sling.api.resource.ValueMap;
 import org.apache.sling.caconfig.ConfigurationBuilder;
+import org.apache.sling.models.annotations.injectorspecific.Self;
 
 import com.adobe.cq.commerce.core.components.datalayer.CategoryData;
+import com.adobe.cq.wcm.core.components.models.Component;
 import com.adobe.cq.wcm.core.components.models.datalayer.AssetData;
 import com.adobe.cq.wcm.core.components.models.datalayer.ComponentData;
+import com.adobe.cq.wcm.core.components.util.ComponentUtils;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 
 public abstract class DataLayerComponent {
-    public static final String ID_SEPARATOR = "-";
+    public static final String ID_SEPARATOR = ComponentUtils.ID_SEPARATOR;
```

Comment:
Why redefining the const instead of using `ComponentUtils.ID_SEPARATOR` directly via static import?

Reply:
The ID_SEPARATOR is used at more other places in the codebe in sub-classes of DataLayerComponnet. If you define it here you don't need any static import in many other files. In fact the constant was defined earlier, I've just aligned the definition with ComponentUtils.

Reply:
I know the constant was defined earlier and it was wrong. So instead of having the same constant twice I would prefer to remove it and only use `ComponentUtils.ID_SEPARATOR`. The static import should not hurt.

Reply:
+1 - this is more explicit.

Reply:
I've inlined the ID_SEPARATOR constant.